src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -251,20 +283,19 @@ func (l *commitLog) Open() error {
 	defer l.closedState.Unlock()
 
 	// Open the buffered commit log writer
-	if _, err := l.openWriter(); err != nil {
+	if _, err := l.openWriters(); err != nil {
 		return err
 	}
 
 	// Sync the info header to ensure we can write to disk and make sure that we can at least
 	// read the info about the commitlog file later.
-	if err := l.writerState.writer.Flush(true); err != nil {
+	if err := l.writerState.primaryWriter.writer.Flush(true); err != nil {
+		return err
+	}
+	if err := l.writerState.secondaryWriter.writer.Flush(true); err != nil {
```

Comment:
nit: Perhaps you could put them both in an array so we don't forget to do any operation to just one commit log?

i.e.

```go
type writerState struct {
  primaryWriter asyncResettableWriter
  secondaryWriter asyncResettableWriter
  writers []commitLogWriter
  activeFiles persist.CommitLogFiles
}


// then

for _, writer := range l.writerState.writers {
  if err := writer.Flush(true); err != nil {
    return err
  }
}
```